SaveData/R/SaveData.R

```diff
@@ -0,0 +1,155 @@
+#' Save and summarize data
+#'
+#' @param df Data to be saved
+#' @param key Defines key to dataset. Must be non-missing and jointly unique.
+#' @param outfile Path to saved data file. Possible file extensions are .csv, .dta, RData, and .RDS. If no file extension is provided, the data is saved as .RDS.
+#' @param logfile Path to logfile. File contains standard summary statistics for numerical variables and displays variable types for all variables. If no path is specified, the log file is saved to the same directory as the output file. Type logfile = FALSE to not output a logfile.
+#' @param appendlog If TRUE, an existing log file of the same name will not be overwritten but the information appended. Default is TRUE.
```

Comment:
Sorry I didn't quite get the logic of setting the default of `appendlog` to be TRUE. I would imagine people don't want to append the log file but want them to be overwritten.

Reply:
This is analogous to the gslab_stata [`save_data`](https://github.com/gslab-econ/gslab_stata/blob/e148ca63ae532fffc45121fdfe0d9b593f9bed29/gslab_misc/ado/save_data.ado) command; I agree I find it less intuitive than overwriting by default. Would tentatively vote to keep it parallel to Stata but let me know if you think otherwise!

Reply:
Got it. I'm fine with keeping the default as TRUE.